### ISSUE TITLE
FQN is not reported in property annotation

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#258]
Example script:
```php
<?php

namespace Acme;

class Foo
{
    /**
     * @var \Doctrine\DBAL\Connection
     */
    public $conn;
}

$conn = new \Doctrine\DBAL\Connection();
```
Actual output:
```
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 13 | ERROR | [x] Class \Doctrine\DBAL\Connection should not be referenced via a fully
    |       |     qualified name, but via a use statement.
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------
```
Expected output: line 8 should be reported as containing FQN as well.

Please see https://github.com/doctrine/dbal/pull/3002#discussion_r165686652 for the reference.

**Comment by kukulich** [comment]
Hmm, the sniff does not check any doccomment but it should be possible very easy. I hope :)

**Comment by kukulich** [comment]
@morozov Could you please try https://github.com/slevomat/coding-standard/commit/c73280903ece590687d0d9abdddb8e8efdb017ca ?

**Comment by morozov** [comment]
Thanks @kukulich. I checked out the commit above and enabled `searchAnnotations` for `SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly` in our [`ruleset.xml`](https://github.com/doctrine/coding-standard/blob/0f48be4c09ef08e9c0cff406d8428d65de628362/lib/Doctrine/ruleset.xml#L109). Still same output.

Most likely, I'm just doing something wrong. @Majkl578 could you please check this?

**Comment by Majkl578** [comment]
Testing on ORM, seems to work, or at least report errors like:
```
FILE: lib/Doctrine/ORM/Query/Exec/AbstractSqlExecutor.php
--------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------------------------------
 56 | ERROR | [x] Class \Doctrine\DBAL\Driver\Statement should not be referenced via a fully qualified name, but via a use statement.
```

For [`@return here`](https://github.com/doctrine/doctrine2/blob/eef6308b1925ec2d98e1d78a8f89962ff2d156d7/lib/Doctrine/ORM/Query/Exec/AbstractSqlExecutor.php#L56).

To enable, require `slevomat/coding-standard: dev-master as 4.3.99` and just add:
```xml
            <property name="searchAnnotations" type="boolean" value="true"/>
```
and delete cache file `.phpcs-cache`.

**Comment by morozov** [comment]
Clearing cache worked for me. Thanks.
